pypeman/contrib/http.py

```diff
@@ -183,7 +183,12 @@ def generate_request_url(self, msg):
                     "This can be fixed using JsonToPython node before your "
                     "RequestNode")
                 raise
-        return self.url % url_dict
+        try:
+            request_url = self.url % url_dict
+        except Exception as exc:
+            logger.error("cannot create url %r with args %r", self.url, url_dict)
```

Comment:
does `url_dict` only contains strings, ints, floats, bools, lists of below or dicts of below.

If not potential pickle problems and it's better to 

logger.error("cannot create url %r with args %s", self.url, repr(url_dict))

Reply:
changed